Examples/Scripts/Python/propagation.py

```diff
@@ -63,6 +63,14 @@ def runPropagation(trackingGeometry, field, outputDir, s=None, decorators=[]):
         )
     )
 
+    s.addWriter(
```

Comment:
Wondering if this could be conditional on whether or not `sterileLogger` is enables in the `PropagationAlgorithm::Config`.